data/helpers/patients.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../dbConfig');
+
+module.exports = {
+    getPatient,
+    getHistory,
+};
+
+function getPatient(id) {
+    // Patient ID
+    return db('patients').where({ id });
+}
+
+function getHistory(id) {
+    // Patient ID
+    return db('patients')
+    .where({ 'patients.id': id })
+    .join('patient_immunizations as PI', { 'patients.id': 'PI.patientId' })
+    .join('immunizations', { 'immunizations.id': 'PI.immunizationId' })
+    .select(
+    'immunizations.name as immunization_name',
+    'PI.appointmentDate as appointment_date'
```

Comment:
Might be a good idea to also give out the immunization id, just in case the FE wants to use it to categorize records by vaccination.